compiler/driver.c

```diff
@@ -149,32 +164,55 @@ static void read_options(struct options * o, int argc, char * argv[]) {
                 o->widechars = false;
                 continue;
             }
-#ifndef DISABLE_JAVA
-            if (eq(s, "-p") || eq(s, "-parentclassname")) {
-                check_lim(i, argc);
-                o->parent_class_name = argv[i++];
-                continue;
-            }
-            if (eq(s, "-P") || eq(s, "-Package")) {
-                check_lim(i, argc);
-                o->package = argv[i++];
-                continue;
-            }
-            if (eq(s, "-S") || eq(s, "-stringclass")) {
-                check_lim(i, argc);
-                o->string_class = argv[i++];
-                continue;
-            }
-            if (eq(s, "-a") || eq(s, "-amongclass")) {
-                check_lim(i, argc);
-                o->among_class = argv[i++];
-                continue;
-            }
+#if !defined(DISABLE_JAVA) || !defined(DISABLE_CSHARP)
+			if (eq(s, "-p") || eq(s, "-parentclassname")) {
+				check_lim(i, argc);
+				o->parent_class_name = argv[i++];
+				continue;
+			}
+			if (eq(s, "-P") || eq(s, "-Package")) {
+				check_lim(i, argc);
+				o->package = argv[i++];
+				continue;
+			}
+			if (eq(s, "-S") || eq(s, "-stringclass")) {
+				check_lim(i, argc);
+				o->string_class = argv[i++];
+				continue;
+			}
+			if (eq(s, "-a") || eq(s, "-amongclass")) {
+				check_lim(i, argc);
+				o->among_class = argv[i++];
+				continue;
+			}
 #endif
```

Comment:
Please avoid whitespace changes like these - they make patches harder to review, and make tools like `git blame` less useful.  Currently the codebase is mostly tab-free (which I think should probably be the policy) but where tabs the width is 8.